hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestBase.java

```diff
@@ -103,22 +103,26 @@ private static String getHiveConsoleCommandFile(String commandFile, String addit
     if (additionalVar != null) {
       builder.append(" --hivevar " + additionalVar + " ");
     }
-    return builder.append(" -f ").append(commandFile).toString();
+    builder.append(" -f ").append(commandFile);
+    System.out.println("Hive command : " + builder.toString());
```

Comment:
can we clean up these s.o.p?

Reply:
this was useful in general while checking the Integ tests log output. If not for this, it wasn't easy to understand where the tests fails if tests hung or terminated half way if not for explicit exception.

Reply:
no I meant, why not log.info() or something?

Reply:
We should look into why these thing are not being logged now. they showed up for sure before

Reply:
you are right. I do see all commands are already logged. my bad.